src/components/Icon/tests/Icon.test.tsx

```diff
@@ -46,6 +64,14 @@ describe('<Icon />', () => {
       expect(element.find('img')).toHaveLength(1);
     });
 
+    it('spits out a console warning when rendering an untrusted SVG', () => {
+      const spy = jest.spyOn(global.console, 'warn');
+      const svg =
+        "<svg><path d="M10 10"/></svg>";
```

Comment:
```suggestion
        "<svg><path d='M10 10'/></svg>";
```

Sorry, I suggested the wrong quotes!